libs/src/services/discoveryProvider/index.js

```diff
@@ -490,14 +490,17 @@ class DiscoveryProvider {
     }
 
     let axiosRequest = this.createDiscProvRequest(requestObj)
-
     let response
     let parsedResponse
     try {
       response = await axios(axiosRequest)
       parsedResponse = Utils.parseDataFromResponse(response)
     } catch (e) {
-      console.error(`Failed to make Discovery Provider request: ${e}`)
+      // If Discovery Provider responds with 4xx status code, content is unavailable. Do not retry.
+      if (e.response && e.response.status && (e.response.status / 100) === 4) {
+        return
```

Comment:
I don't think this is right. If someone turns off their discovery provider and it starts spitting 4XXs, the client should probably retry somewhere else, right? Or at least do so until max retries?

Reply:
hm I see. ok sure, we should try other disc provs. I can remove this.